pyre/bench/synth/exec_foreign_globals_gc_root.py

```
@@ -0,0 +1,71 @@
+        a = Idx(i % 5)
+        b = Idx(i % 5 + 3)
+        acc = acc + len(seq[a:b]) + len(lst[a:b]) + len(tup[a:b])
+        i = i + 1
+    print("acc", acc)
+    print(call(lambda: 1))
+
+
+main()
+'''
+
+exec(compile(WORKLOAD, __file__, "exec"), {"__name__": "__main__", "__file__": __file__})
```

Ruff (0.16.1)

[error] Use of `exec` detected

(S102)